bundles/CoreBundle/src/EventListener/Frontend/ElementListener.php

```diff
@@ -136,8 +136,9 @@ private function handleAdminUserDocumentParams(Request $request, ?Document $docu
         }
 
         // editmode document
+        $isPimcoreStudio = $request->query->getBoolean('pimcore_studio');
         if ($this->editmodeResolver->isEditmode($request)) {
-            $document = $this->handleEditmode($document, $user, $request->getSession());
+            $document = $this->handleEditmode($document, $user, $request->getSession(), $isPimcoreStudio);
         }
 
```

Comment:
Does studio still use `pimcore_preview` and `pimcore_version` query string params?

Reply:
Yes but we already use `pimcore_studio_preview` for the same purpose of the object preview, so let's call it the same. Would make sense to check it, right!